pages/clusters-for-kafka/faq.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Clusters for Apache Kafka® FAQ
+description: Get answers to all your questions about Scaleway Clusters for Apache Kafka®
+dates:
+  validation: 2025-08-20
+  posted: 2025-08-20
+productIcon: KafkaProductIcon
+noindex: true
+---
+
+<Message type="note">
+Clusters for Apache Kafka® is currently in [Private Beta](https://www.scaleway.com/en/betas/) and available to selected testers only.
```

Comment:
Sometimes you wrote "Cluster for Apache Kafka" and sometimes "Clusters for Apache Kafka" as product name - should it be "Clusters" everywhere?